contrib/themes/README.md

```diff
@@ -0,0 +1,3 @@
+## Ingrid
+
+![ingrid.toml](screenshots/ingrid.png)
```

Comment:
I think we should keep the themes in this folder, but the screenshot previews should be on the github wiki. That way it doesn't add to the repository size when cloning

Reply:
That's a good idea. I don't seem to have access to the wiki though, could you make the wiki page?

Reply:
https://github.com/helix-editor/helix/wiki/Themes

Reply:
I still can't seem to edit that wiki page?

Reply:
Need to wait for @archseer to enable that.

Reply:
I just enabled public write access now, go ahead!